### ISSUE TITLE
dino on pinephone/sxmo doesn't use its mobile mode.

### ISSUE THREAD

**Issue by norayr** [dino/dino#1438]
in mobile mode dino shows only the part of the window where users exchange messages, but user list is hidden.
in order to go to user list the user must press the button in the left upper corner.

on pinephone/sxmo, however, dino doesn't recognize that the screen is mobile phone's screen and shows buth panes, the user list and the messages window.

that is very inconvenient because both are very narrow and not very usable.

please test dino on pinephone/sxmo.

**Comment by fiaxh** [comment]
Are you maybe using an older Dino version on the Pinephone? Dino 0.3 and earlier are not adapted for mobile at all.

**Comment by norayr** [comment]
yes, i think if the conversation has the image then it works, i can see the button in the left to go to contacts pane and only the conversation pane is visible. othewrise both panes are visible.

**Comment by norayr** [comment]
another update. i have been suggested on the forums to increase the dpi. 

so when i do

```
xrandr --output DSI-1 --scale 0.95x0.95
```

nothing changes

when i do 

```
xrandr --output DSI-1 --scale 0.90x0.90
```

most of the times dino works correctly, but the svkbd keyboard becomes almost unusable. dino doesn't work correctly all the time, but most of the time.

when i do

```
xrandr --output DSI-1 --scale 0.85x0.85
```

then dino always works correctly but the keyboard becomes unusable and many keys not visible.

**Comment by norayr** [comment]
ok i also decreased font sizes in .gtk.ini and sxmo config, and increased dpi so i think it is solved for me?
